paddle/fluid/framework/details/eager_deletion_op_handle.h

```diff
@@ -37,14 +40,16 @@ class EagerDeletionOpHandle : public OpHandleBase {
 
   std::string Name() const override;
 
+  Priority GetPriority() const override { return kHighest; }
+
  protected:
   void RunImpl() override;
 
  private:
   void ClearGarbages(std::deque<std::shared_ptr<memory::Allocation>> *garbages);
 
   const Scope *scope_;
-  std::unordered_set<std::string> var_names_;
+  std::vector<std::string> var_names_;
```

Comment:
Why do you replace `std::unordered_set` with `std::vector` here.

Reply:
It does not matter. Just for faster transversing of `var_names_`.